spec/index.bs

```diff
@@ -7346,9 +7357,11 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
     Return `true` if all of the following conditions are met, and `false` otherwise:
 
     - |descriptor|.{{GPUProgrammableStage/module}} must be a [=valid=] {{GPUShaderModule}}.
-    - |descriptor|.{{GPUProgrammableStage/module}} must contain
-        an entry point, for shader stage |stage|,
-        named |descriptor|.{{GPUProgrammableStage/entryPoint}}.
+    - If |descriptor|.{{GPUProgrammableStage/entryPoint}} is not `null`,
+        |descriptor|.{{GPUProgrammableStage/module}} must contain an entry point, for shader stage
+        |stage|, named |descriptor|.{{GPUProgrammableStage/entryPoint}}. Otherwise,
+        |descriptor|.{{GPUProgrammableStage/module}} must contain exactly one entry point for shader
+        stage |stage|.
```

Comment:
```suggestion
    - If |descriptor|.{{GPUProgrammableStage/entryPoint}} is not `null`:

        - |descriptor|.{{GPUProgrammableStage/module}} must contain exactly one
            entry point for shader stage |stage| with a name equalling
            |descriptor|.{{GPUProgrammableStage/entryPoint}}.

        Otherwise:

        - |descriptor|.{{GPUProgrammableStage/module}} must contain exactly one
            entry point for shader stage |stage|.
```